### DISCUSSION TITLE
Rainbowkit for react native

### DISCUSSION THREAD

**Discussion by maxmandia** [rainbow-me/rainbowkit#708] [Ideas]
hi team - any plans to add rainbowkit support for react native? would be awesome :)

**Comment by META-DREAMER** [comment]
+1, would be excellent especially if it used the new WalletConnect 2.0 integrations for RN: https://docs.walletconnect.com/2.0/javascript/guides/react-native

**Reply by adrianotadao** [reply]
hey @META-DREAMER are you using walletconnect 2.0 yet? If so, does it work well for react native? I'm a little bit confused how to use it and metamask wallet to connect to my app.

**Reply by META-DREAMER** [reply]
Haven’t heard success integrating it with wagmi.sh yet, but the basic example was working

**Comment by mobiman1** [comment]
+1 we would definitely like to add RainbowKit to our connect options once it becomes available for React Native
